### PR TITLE
Added formatting option.

### PR DESCRIPTION
**Pull request by KilianDargel**
Now utilises the tshark -F flag. Note that this also requires to change the default filename to dump.pcap.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,8 @@ ENV FILTER=""
 ENV IFACE="any"
 ENV MAXFILESIZE="1000"
 ENV MAXFILENUM="10"
-ENV FILENAME="dump"
+ENV FILENAME="dump.pcap"
+ENV FORMAT="pcap"
```
Comment (tlnd) [inline Dockerfile]: As default capture format is `pcapng` at present, I'd rather make that the default, which can be overwritten to enforce `pcap` format, if needed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+## v1.0.2- Feature: Added formatting option
```
Comment (tlnd) [inline CHANGELOG.md]: You add a feature, hence increase the minor version -- not the patch version.

File: Dockerfile (hunk 1 above)
Comment (KilianDargel) [reply Dockerfile]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (KilianDargel) [reply CHANGELOG.md]: Done.

File: README.md [hunk 3]
```diff
@@ -41,6 +42,10 @@ The `FILENAME` variable sets the filename that is used. The default value is
 information). To dump on multiple interfaces simply add more interfaces to this 
 variable seperated by a whitespace (e.g. "eth0 eth1").
 
+The `FORMAT` variable sets the format of the written file. Use the variable `dump.pcap` as
+the `FILENAME` option and `pcap` for `FILENAME` to use the common pcap-format. Other formats are described in
+the [official documentation](https://www.wireshark.org/docs/man-pages/tshark.html).
+
```
Comment (mgumz) [inline README.md:48]: this section here is rather confusing. why is `dump.pcap` a variable? isnt it the value of a variable?
i shall use `dump.pcap` for `FILENAME` and also `pcap` for `FILENAME`?

in generell: why dont you just use something like `ADDITIONAL_FLAGS` to transport any flag someone likes down to `dumpcap` and let users use "-F pcap" if they desire it. document `how to change from pcapng to pcap format` here for sure, but don't make the script more complex then it needs to be

Comment (KilianDargel) [reply README.md:48]: >i shall use dump.pcap for FILENAME and also pcap for FILENAME?
Yes, using the default FILENAME `dump` and using the value `pcap` as the `FORMAT` will not result in .pcap files. The FILENAME also has to be renamed to `dump.pcap` making this a bit confusing to explain

Comment (tlnd) [reply README.md:48]: Yes, ok. But you declare two times to set `FILENAME` -- the second time you probably refer to `FORMAT`.

File: run.sh [hunk 4]
```diff
@@ -35,4 +35,4 @@ then
   BUFFEROPTS="$BUFFEROPTS -b duration:$DURATION"
 fi
 
-/usr/bin/tshark $BUFFEROPTS -w "/data/$FILENAME" -f "$FILTER" $INTERFACES 
+/usr/bin/tshark $BUFFEROPTS -w "/data/$FILENAME" -f "$FILTER" $INTERFACES -F $FORMAT
```
Comment (tlnd) [inline run.sh:38]: Are you sure this will run if `FORMAT` is not defined?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
+## v1.1.0- Feature: Added formatting option
```
Comment (tlnd) [inline CHANGELOG.md]: Please fix line break

File: run.sh (hunk 4 above)
Comment (tlnd) [reply run.sh:38]: No, it doesn't:

```
$ docker run -ti --rm docker-pcap
/usr/bin/tshark: option requires an argument: F
```

**Pull request merged**